evm-adapters/testdata/CheatCodes.sol

```diff
@@ -209,6 +211,40 @@ contract CheatCodes is DSTest {
         assertEq(string(newCode), string(n_code));
     }
 
+    function testExpectRevert() public {
+        ExpectRevert target = new ExpectRevert();
+        hevm.expectRevert("Value too large");
+        target.stringErr(101);
+        target.stringErr(99); 
+    }
+
+    function testExpectCustomRevert() public {
+        ExpectRevert target = new ExpectRevert();
+        bytes memory data = abi.encodePacked(bytes4(keccak256("InputTooLarge()")));
```

Comment:
Maybe do this with `abi.encodeWithSignature` (think that's what @transmissions11 linked to from the solidity docs)?

Reply:
yup the blogposts do it like so: `abi.encodeWithSignature("InsufficientBalance(uint256,uint256)", balance[msg.sender], amount)`

https://blog.soliditylang.org/2021/04/21/custom-errors/

Reply:
doesnt work :( encodeWithSignature for an error that takes nothing adds bytes to the end
```
[FAIL. Reason: Error data != expected error data: 0x7f7be572 != 0x7f7be57200000000000000000000000000000000000000000000000000000000000000200000000000000000000000000000000000000000000000000000000000000000] 
```

Reply:
Wow lol